lib/src/chart/line_chart/line_chart_renderer.dart

```diff
@@ -10,14 +10,21 @@ import 'package:flutter/material.dart';
 
 /// Low level LineChart Widget.
 class LineChartLeaf extends LeafRenderObjectWidget {
-  const LineChartLeaf({Key? key, required this.data, required this.targetData})
-      : super(key: key);
+  const LineChartLeaf({
```

Comment:
These changes can be reverted. Please revert them to keep the PR as clean as possible.